solr/core/src/test/org/apache/solr/handler/admin/ShowFileRequestHandlerTest.java

```diff
@@ -100,18 +100,15 @@ public String getWriterType() {
           }
 
           @Override
-          public NamedList<Object> processResponse(InputStream body, String encoding) {
-            try {
-              if (body.read() >= 0) readFile.set(true);
-            } catch (IOException e) {
-              throw new RuntimeException(e);
-            }
+          public NamedList<Object> processResponse(InputStream body, String encoding)
+              throws IOException {
+            if (body.read() >= 0) readFile.set(true);
             return null;
           }
 
           @Override
-          public NamedList<Object> processResponse(Reader reader) {
-            throw new UnsupportedOperationException("TODO unimplemented"); // TODO
+          public Set<String> getContentTypes() {
+            return Set.of(); // don't enforce
```

Comment:
if you don't know about the previous `UOE` usage, then the comment doesn't seem useful.   I think it isn't adding value.

Reply:
The file diff is confusing here because what was removed and added have nothing to do with each other.  Two things are happening here:
(1) removed processResponse(Reader) from ResponseParser and thus the implementation in this file need not implement (it was compelled to before)
(2) getContentTypes is now abstract and so all implementations must implement